.github/workflows/tests.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches: master
   pull_request:
+env:
+  HOMEBREW_NO_INSTALL_FROM_API: 1
```

Comment:
Would we rather set this, or just `brew untap homebrew/core` here?

The issue is that we have a stale checkout of homebrew/core which sometimes breaks the tests, so we either need to update it by setting this or avoid it (maybe, not sure) by untapping.

CC @MikeMcQuaid @Bo98